### PR TITLE
fix: remove default role manager domain map

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

### CODE REVIEW

**Comment by closetool** [comment]
@hsluoyz plz review

File: rbac/default-role-manager/role_manager.go [hunk 1]
```diff
@@ -24,13 +25,15 @@ import (
 	"github.com/casbin/casbin/v2/util"
 )
 
-const defaultDomain string = "casbin::default"
+const defaultDomain string = "casbin_default"
```
Comment (hsluoyz) [inline rbac/default-role-manager/role_manager.go:28]: Why `defaultDomain` is not empty string?

I think a global user alice should be ``::alice``

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@closetool can you post the comparison data of our benchmark test: https://casbin.org/docs/en/benchmark ?

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
The performance diff:

image

**Pull request merged**